tools/build_node.js

```diff
@@ -41,23 +41,18 @@ async function buildNodeHighlightJS() {
 }
 
 async function buildPackageJSON() {
-  const CONTRIBUTOR = /^- (.*) <(.*)>$/
-
-  let authors = await fs.readFile("AUTHORS.txt", {encoding: "utf8"})
-  let lines = authors.split(/\r?\n/)
-  let json = require("../package")
-  json.contributors = lines.reduce((acc, line) => {
-    let matches = line.match(CONTRIBUTOR)
-
-    if (matches) {
-      acc.push({
-        name: matches[1],
-        email: matches[2]
-      })
-    }
-    return acc;
-  }, []);
-  await fs.writeFile(`${process.env.BUILD_DIR}/package.json`, JSON.stringify(json, null, '   '));
+  const CONTRIBUTOR = /^- ([^<(]+( ?<.+>)?( ?\(http.+\))?)/;
+
+  const authors = await fs.readFile("AUTHORS.txt", { encoding: "utf8" });
+  const lines = authors.split(/\r?\n/);
+  const packageJson = require("../package");
+
+  packageJson.contributors = lines
+    .map(line => line.match(CONTRIBUTOR))
+    .filter((matches) => matches)
+    .map((matches) => matches[1]);
+
+  await fs.writeFile(`${process.env.BUILD_DIR}/package.json`, JSON.stringify(packageJson, null, 2));
```

Comment:
Does this mean 2 spaces?  Was the change from 3 spaces to 2 here intentional?

Reply:
Yes. 
npm itself formats the `package.json` to two spaces when modifying, so was aligning with that.